backend/src/main.ts

```diff
@@ -14,7 +14,7 @@ async function bootstrap(): Promise<void> {
     .build();
 
   const document = SwaggerModule.createDocument(app, config);
-  fs.writeFileSync("./swagger-spec.json", JSON.stringify(document));
+  fs.writeFileSync("./swagger-spec.json", JSON.stringify(document, null, 2));
```

Comment:
Good idea - I've already wondered if there's some solution not to have a one-liner in swagger-spec.json :)